src/data/team-data.js

```diff
@@ -33,7 +33,8 @@ export default [
   },
   {
     title: 'Stephen Drollinger',
-    summary: 'Stephen Drollinger is ...',
-    content: 'Stephen Drollinger is ...',
+    github: 'https://github.com/Valimere',
+    summary: 'Stephen Drollinger is a software engineer at Ironnet Cybersecurity working on the Detection Development turning new analytics into production code. ',
```

Comment:
Stephen Drollinger is a software engineer at Ironnet Cybersecurity, working on Detection Development, turning new analytics into production code.

Reply:
> Stephen Drollinger is a software engineer with a strong DevOps background. He currently works at IronNet Cybersecurity on Detection Development, turning new analytics into production code.